spk/dnscrypt-proxy/src/update-blocklist.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Add randomness when running cron job (0-30 minutes)
+# Be nice to servers and don't update at exactly midnight at the same time
+# shellcheck disable=SC2039
+[ -n "$RANDOM" ] && sleep $((RANDOM % 1800))
+
+cd /var/packages/dnscrypt-proxy/target/var/ || exit
+/usr/bin/env python generate-domains-blacklist.py > blacklist.txt.tmp && mv -f blacklist.txt.tmp blacklist.txt
```

Comment:
Still some color here... Please rename.

Reply:
Done. But please not that none of this user facing unless you ssh into your Synology.